docker-compose.yml

```diff
@@ -1,6 +1,73 @@
 version: '3'
 
 services:
+    airflow-flower:
```

Comment:
Should probably prefix these all with `article-store-`

Reply:
`article-store-airflow-*` ?